### PR TITLE
docs: Updated README.md

### PR DESCRIPTION
**Pull request by Souheil-Yazji**
docs: Updated README.md.

### CODE REVIEW

**Review by DwangDistracted — APPROVED** [review]

**Review by Weebian — APPROVED** [review]
looks good

File: README.md [hunk 1]
```diff
@@ -1 +1,22 @@
-# Mini-Shopify
\ No newline at end of file
+# Mini-Shopify
+## About
+This project aims to create a platform, inspired by shopify, which allows signed-in users to create shops and any web user to browse and purchase products from said shops. <br /> <br />
```
Comment (dershao) [inline README.md:3]: capital shopify

File: README.md [hunk 2]
```diff
@@ -1 +1,22 @@
...
+## Development Stack
+1. FrontEnd: Vue.JS + BootstrapVue
+2. Backend: Spring with REST endpoints
+3. Database: PostgreSQL 
+### [Travis-CI](https://travis-ci.com/github/mpfan/Mini-Shopify)
+### [Heroku App](https://sh0p1fy.herokuapp.com/)
+<br /> 
+
+## Contribution Convention
+Create a feature branch using main as base. Use the following link for [Commit Convention](https://www.conventionalcommits.org/en/v1.0.0/). All contributions to the project should pretain to an [Issue](https://github.com/mpfan/Mini-Shopify/issues), and should be logged in the Kanban project. Individual contributions to the project should be made through Pull Requests into the Main branch. Issues should be updated with the appropriate resolving pull request.
```
Comment (dershao) [inline README.md:20]: pretain --> pertain

File: README.md [hunk 3]
```diff
@@ -1 +1,22 @@
...
\ No newline at end of file
+# Mini-Shopify
+## About
+This project aims to create a platform, inspired by shopify, which allows signed-in users to create shops and any web user to browse and purchase products from said shops. <br /> <br /> 
+## Authors
+- David Wang
+- Derek Shao
+- Hoang Bui
+- Michael Fan
+- Souheil Yazji
```
Comment (mpfan) [inline README.md:9]: Order by last name.

**Pull request merged**